src/OV/System.hpp

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright The XCSoar Project
+
+#pragma once
+
+enum class SSHStatus {
+  ENABLED,
+  DISABLED,
+  TEMPORARY,
+};
+
+[[gnu::const]]
```

Comment:
this attribute is wrong. Do you know the difference between "const" and "pure"?

Reply:
Not sufficiently apparently. After educating myself a bit, I'm not sure if either fits here. I'd remove that one entirely.

Reply:
"pure" would be correct here - the return value can change if global state changes, but the return value will stay the same if you call it twice right way. "const" ignores global state.